### PR TITLE
Dashboard updates

### LINKED ISSUE #343: Dashboard Request: Make "Test Specification" and “Environment Details" feel like tabs.
#### Mockups

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by asubedy**
**Description**

This PR fixes #343 

Uses the three page layout.

link to all performance profile: `/dashboard/all`

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by asubedy** [comment]
Ok wait do-not merge pagination on deploy is not working, It is working perfectly in my local build. Need to look into this.

**Review by warunicorn19 — CHANGES_REQUESTED** [review]
- Could you add an empty state when there's no results in the table? 

Screenshot 2022-06-13 at 10 43 44 PM

- Also, I think the chevron should be sticky. Otherwise it gets lost when the drawer is closed.
Screenshot 2022-06-13 at 10 44 29 PM

- Also probably a different color for the sidebar? Maybe a lighter shade of the same color.

File: docs/assets/js/profiles.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+pages = document.getElementById("pages")
+
+let test_type = cardRow.attributes[2].value
+
+console.log(test_type)
+
+// if(window.location.pathname == "/dashboard/" || window.location.pathname == "/dashboard" ){
+//   reloadFunction(1)
+// }
+console.log(window.location.pathname.includes("#"))
```
Comment (warunicorn19) [inline docs/assets/js/profiles.js:11]: console logs can be removed.

**Comment by asubedy** [comment]
I guess this is ready to merge @leecalcote @warunicorn19 ? Any more changes required?

**Pull request merged**